code_samples/api/commerce/src/Command/ShipmentCommand.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types=1);
+ 
+namespace App\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Ibexa\Contracts\Shipping\ShipmentServiceInterface;
+use Ibexa\Contracts\Shipping\Value\ShipmentCreateStruct;
+use Ibexa\Contracts\Shipping\Value\ShipmentMetadataUpdateStruct;
+use Ibexa\Contracts\Shipping\Value\ShipmentQuery;
+use Ibexa\Contracts\Shipping\Value\ShipmentUpdateStruct;
+use Ibexa\Core\Repository\Permission\PermissionResolver;
+
+
+final class ShipmentCommand extends Command
+{
+    private PermissionResolver $permissionResolver;
+
+    private UserService $userService;
+
+    private ShippingServiceInterface $shipmentService;
+
+    public function __construct(
+      PermissionResolver $permissionResolver, 
+      UserService $userService,
+      ShippingServiceInterface $shipmentService
+      ) {
+        $this->shipmentService = $shipmentService;
+        $this->permissionResolver = $permissionResolver;
+        $this->userService = $userService;
+
+        parent::__construct('doc:shipment');
+    }
+
+    public function configure(): void
+    {
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): 
+    {
+        $currentUser = $this->userService->loadUserByLogin('admin');
+        $this->permissionResolver->setCurrentUserReference($currentUser);
+
+        // Get a single shipment
+        $id = 1;
+        $shipment = $this->shipmentService->getShipment($id);
+
+        $output->writeln(sprintf('Shipment %d has status %s', $id, $shipment->getStatus()));
+
+        // Get a single shipment by identifier
+        $shipmentIdentifier = '4ac4b8a0-eed8-496d-87d9-32a960a10629';
+        $shipment = $this->shipmentService->getShipmentByIdentifier($shipmentIdentifier);
+
+        $output->writeln(sprintf('Your shipment has status %s', $shipment->getStatus()));
+
+        // Query for shipments
+        $shipmentCriterions = [
+            new ShippingMethodCriterion(1),
```

Comment:
`use Ibexa\Contracts\Checkout\Shipment\Query\Criterion\ShippingMethod;` import is needed and then we can use it here 

```
new ShippingMethod(1),
```
or 
`use Ibexa\Contracts\Checkout\Shipment\Query\Criterion\ShippingMethod as ShippingMethodCriterion;`
and then 
```
new ShippingMethodCriterion(1),
```